pkgs/top-level/python-packages.nix

```diff
@@ -8626,6 +8626,8 @@ let
       ${self.python.executable} runtests.py
     '';
 
+    __impureHostDeps = optionals stdenv.isDarwin [ "/usr/lib/libm.dylib" ];
+
```

Comment:
it's to get llvmlite build, it was a broken (on darwin) dependency for scikit-image; numba is still broken on darwin